client/client.go

```diff
@@ -364,6 +364,22 @@ func (c *Client) ListUsers() (*http.Response, error) {
 	return c.get("/user/list", nil)
 }
 
+// EnableTotp enables Totp for a given user
+func (c *Client) EnableTotp(username, password string) (*http.Response, error) {
+	return c.post("/user/totp/enable", &common.UserRequest{
+		Username: username,
+		Password: password,
+	})
+}
+
+// DisableTotp enables Totp for a given user
+func (c *Client) DisableTotp(username, password string) (*http.Response, error) {
+	return c.post("/user/totp/disable", &common.UserRequest{
+		Username: username,
+		Password: password,
+	})
```

Comment:
@kieranhalliday We're going to need users to present the TOTP as well when they're disabling TOTP.